src/server/tsconfig.library.json

```diff
@@ -93,6 +93,7 @@
         "../services/refactorProvider.ts",
         "../services/codefixes/addMissingInvocationForDecorator.ts",
         "../services/codefixes/annotateWithTypeFromJSDoc.ts",
+        "../services/codefixes/convertPromisesToAwaitAndAsync.ts",

```

Comment:
Minor nit: But this file is inserted into slightly different locations in this and the prior two tsconfig files. Looks like it's supposed to be alphabetical. (So it would seem the src/server/tsconfig.json ordering is correct out of the 3).